files/opt/dash-button/scripts/providers/README.md

```diff
@@ -0,0 +1,83 @@
+## Configure Providers
+Below you can find detailed information and configurations for supported providers.
+
+Please be aware that the provider must have the same name as a js file in the providers folder in order to match the provider.
+
```

Comment:
I'd split that file and add it to each provider's folder. Then one can maintain a provider by only changing files in the directory.

Also you could add a list of supported providers with links to their README files in the main README.